pkg/utils/secrets/manager/generate.go

```diff
@@ -206,7 +206,13 @@ func (m *manager) generateBundleSecret(ctx context.Context, config secretsutils.
 		if c.SigningCA == nil {
 			certs := [][]byte{secrets.current.obj.Data[secretsutils.DataKeyCertificateCA]}
 			if secrets.old != nil {
-				certs = append(certs, secrets.old.obj.Data[secretsutils.DataKeyCertificateCA])
+				valid, err := m.isStillValid(secrets.old.obj)
+				if err != nil {
+					return fmt.Errorf("failed validating old secret: %w", err)
```

Comment:
I think adding the name of the old secret would be helpful when resolving the error